### ISSUE TITLE
Shaman totems buff still not showing

### ISSUE THREAD

**Issue by Romeus83** [Resike/Z-Perl#46]
Hi, Shaman totems are still not displayed..is there is any resolution to this or can anyone advice? Much appreicated. Thanks!

**Comment by Romeus83** [comment]
Hi any response to this would be nice. Thanks

**Comment by Resike** [comment]
I can't reproduce this, have you tried with other all addons turned off?

**Comment by Romeus83** [comment]
Yes. same issue with all addons turned off. Or can you guide how to on it?
